scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/task/TaskStateImpl.java

```diff
@@ -0,0 +1,73 @@
+package org.ow2.proactive.scheduler.task;
+
+import java.util.List;
+
+import org.ow2.proactive.scheduler.common.task.TaskInfo;
+import org.ow2.proactive.scheduler.common.task.TaskState;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+
+public class TaskStateImpl extends TaskState {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2160)